.github/dependabot.yml

```diff
@@ -70,6 +70,32 @@ updates:
     allow:
       - dependency-name: "@saucelabs/*"
         dependency-type: all
+  - package-ecosystem: npm
```

Comment:
Shouldn't we use multi-directory config for this:
- https://github.blog/changelog/2024-06-25-simplified-dependabot-yml-configuration-with-multi-directory-key-directories-and-wildcard-glob-support/
- https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directories

It's probably easier to maintain

Reply:
This cleans it up very nicely. Thanks for the tip.